hanggame/locales/en_CA/LC_MESSAGES/hanggame.po

```diff
@@ -1,195 +1,248 @@
-# Translations template for PROJECT.
```

Comment:
Where is the procedure to translate the app ? Should be documented in the readme (developper part)